source/aggregation-pipeline-builder.txt

```diff
@@ -70,6 +72,8 @@ You can use the |compass-short| Aggregation Pipeline Builder to:
 
 - :ref:`Export a pipeline to Java, Node, C#, and Python 3 <compass-export-pipeline>`
 
+- :ref:`Refreshing Document Counts after Delete Operations <refresh-document-counts-after-deletes>`
```

Comment:
Can we edit this to not use a gerund? Removing the gerund will make this more direct:

"Refresh Document Counts..."